imports/ui/components/CardList.jsx

```diff
@@ -13,6 +13,52 @@ class CardList extends Component {
     this.props.getItems({});
   }
 
+  sortByPricePressed = () => {
+    let items = this.props.items.items;
+
+    if (items.length > 1) {
+      items.sort(function(a, b) {
+        // Function must be inside here or else cannot be used because of scope
+        let convertToKg = unitString => {
+          if (unitString == "per 100g") {
+            return 0.1;
+          } else if (unitString == "per lb") {
+            return 0.4536;
+          } else {
+            return 1;
+          }
```

Comment:
I think a switch statement might look better here instead? 
[https://www.geeksforgeeks.org/switch-vs-else/](url)

According to this, switch statements are usually more efficient compared to nested if-else statements too!

Reply:
if-else is pretty fast, a switch wouldn't be much faster here. But sure, I can change it for readability. Even the geeksforgeeks link say: 

"4. Speed: A switch statement might prove to be faster than ifs provided number of cases are good. If there are only few cases, it might not effect the speed in any case. *Prefer switch if the number of cases are more than 5 otherwise*, you may use if-else too."